tidb-cloud/connect-to-tidb-cluster-serverless.md

```diff
@@ -5,29 +5,57 @@ summary: Learn how to connect to your TiDB Serverless cluster via different meth
 
 # Connect to Your TiDB Serverless Cluster
 
-This document introduces the methods to connect to your TiDB Serverless cluster.
+This document describes how to connect to your TiDB Serverless cluster.
 
 > **Tip:**
 >
 > To learn how to connect to a TiDB Dedicated cluster, see [Connect to Your TiDB Dedicated Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
 
+## Connection methods
+
 After your TiDB Serverless cluster is created on TiDB Cloud, you can connect to it via one of the following methods:
 
-- [Connect via private endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) (recommended)
+- Direct connections
+
+  Direct connections mean the MySQL native connection system over TCP. You can connect to your TiDB Serverless cluster using any tool that supports MySQL connection, such as [MySQL client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html).
+
+- [Data Service (beta)](/tidb-cloud/data-service-overview.md)
+
+  TiDB Serverless provides a Data Service feature that enables you to connect to your TiDB Serverless cluster via an HTTPS request using a custom API endpoint. Unlike direct connections, Data Service accesses TiDB Serverless data via a RESTful API rather than raw SQL.
```

Comment:
```suggestion
  TiDB Cloud provides a Data Service feature that enables you to connect to your TiDB Serverless cluster via an HTTPS request using a custom API endpoint. Unlike direct connections, Data Service accesses TiDB Serverless data via a RESTful API rather than raw SQL.
```